admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -137,3 +139,60 @@
 .nav-bar-overflow .ps__rail-y {
   left: 0 !important;
 }
+
+/**
+ * Fix some RTL problem with `admin-dev/themes/new-theme/scss/pages/stock/stock_page.scss` page
+ * for details, take a look at https://github.com/PrestaShop/PrestaShop/issues/27864 
+ */
+
+.adminstockmanagement .stock-app #search .search-input.tags-input:not(.search-with-icon) {
```

Comment:
Might be stupid but, is stock page the only place where we have such buttons/inputs?

Don't we have something like that on the new product page for example?

Reply:
We probably have this problem for some other pages but it is out of the described issue and it looks fixed in 8.x branches

Considering selector priority, it is hard to make a generic solution from a global `.rtlfix` file when style are defined by a dedicated style page : https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/admin-dev/themes/new-theme/scss/pages/stock/stock_page.scss